Plugins/Public/base_plugin/CoreModule.cpp

```diff
@@ -552,6 +577,20 @@ bool CoreModule::Timer(uint time)
 					ConPrint(L"CoreModule::timer space_obj=%u health=%f\n", space_obj, base->base_health);
 
 			}
+			else
+			{
+				//If they don't eat food, then check if base has money for one repair to pay them salary, if base has not enough of money. then riots. kill 10 crew every 12 hours.
+				if (time % 43200 == 0)
+				{
+					int moneyForOneRepair = CalcMoneyForOneRepair();
+					if (!(base->money > moneyForOneRepair))
```

Comment:
```suggestion
					if (base->money < moneyForOneRepair)
```

Reply:
Almost, it's only to replace oxygen/food/e.t.c with "people do riots/fired" if there's no sufficient money. No extra money is taken here, 10 crew is removed per 12 hours if no money was at a base.

players just would be needed to transport crew if their cash was lower than needed to keep people's salaries.